frontend/src/const/routes.ts

```diff
@@ -298,6 +298,10 @@ export const ROUTE_PATHS = Object.freeze({
     SshCaByIDPage: setRoute(
       "/ssh/$projectId/ca/$caId",
       "/_authenticate/_inject-org-details/_org-layout/ssh/$projectId/_ssh-layout/ca/$caId"
+    ),
```

Comment:
(ignore placement of comment)

Say I want to add some sort of host override for the same SSH user whose login mapping is inherited from a host group. How do I customize it specifically for a host?
<img width="1485" alt="image" src="https://github.com/user-attachments/assets/425d6fcf-0406-45e3-9f73-aa2a95aa4b57" />

Reply:
Just added the ability to override an login user mapping inherited from a host group.